src/components/AnnotationBrowser/AnnotationList.vue

```diff
@@ -126,7 +126,11 @@
                 :key="propertyId"
                 :class="tableItemClass"
               >
-                <span>{{ item[propertyId] }}</span>
+                <span>{{
+                  typeof item[propertyId] === "number"
+                    ? Math.round(item[propertyId] * 100) / 100
```

Comment:
What about _item[propertyId].toFixed(2)_ ?

Reply:
`(42).toFixed(2)` will return `42.00` but using rounding will return `42`
This is just a matter of preference